extensions/panache/hibernate-orm-panache-common/runtime/src/main/java/io/quarkus/hibernate/orm/panache/common/runtime/CommonPanacheQueryImpl.java

```diff
@@ -54,6 +54,8 @@ public void close() {
 
     private Map<String, Map<String, Object>> filters;
 
+    private final String lineSeparator = System.getProperty("line.separator");
```

Comment:
Is it really a good idea to use this? From the original issue, it seems that the new lines are provided by the source code and this won't necessarily match the system line separator? Or am I missing something?

Reply:
This may be difficult to cover all use cases as someone may add “\n” , “\r”, or “\r\n” anywhere and request can comes from a config file.
Maybe it should be done for the 3 possible line separator, or using a "\R": https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#lineending

Reply:
I think that makes sense. Do you want to follow up with that @loicmathieu ?

Reply:
Thinking of weird contrived edge cases, what if someone writes a query like this...?
```sql
select o.id, 'came from Sydney' as comment from Orders o
```

Reply:
@gsmet i'll provide a PR to direclty handle `\n` and `\r` it will be better.
@Felk this kind of edge case cannot be handled as we didn't parse the query. For this, I strongly suggest to not use Panache.